### PR TITLE
extract log likelihood computation

### PR DESCRIPTION
**Pull request by arunge**
I extracted the part for the log likelihood computation. Could you please
- check for correctness, and
- suggest a more detailed documentation:
  ```
  #' @param matrixDiff (matrix) matrix differenc
  #' @param XX (matrix) XX
  #' @param betamcSigma (matrix) betamcSigma
  #' @param smcSmpls (numeric)  sigmas of MC samples
  #' @param sdVar (logical) TRUE if variable standard deviation
  ```

### CODE REVIEW

File: R/01-fitModel.R [hunk 1]
```diff
@@ -428,3 +426,22 @@ cpostX <- function(XX,
   #   max = xobs + sqrt(sigma.obs)
   # ))
 }
+
+#' Get LLog
+#'
+#' Log likelihood computation for (W)AIC
+#'
+#' @param matrixDiff (matrix) matrix differenc
+#' @param XX (matrix) XX
```
Comment (mgross) [inline R/01-fitModel.R]: design matrix

File: R/01-fitModel.R [hunk 2]
```diff
@@ -428,3 +426,22 @@ cpostX <- function(XX,
...
+#' @param betamcSigma (matrix) betamcSigma
```
Comment (mgross) [inline R/01-fitModel.R]: coefficients of varying standard deviation spline

**Comment by mgross** [comment]
@arunge: looks good so far. Have you checked the equality on a single example? If good, then can be merged

**Comment by arunge** [comment]
> @arunge: looks good so far. Have you checked the equality on a single example? If good, then can be merged

@mgross How can I check the results for a single call of `fitPlotRModel()`. `set.seed(123)` is not working here. With each run the result (the samples) will be different. So, I cannot test a single example.

What kind of check do you have in mind?

**Comment by arunge** [comment]
@mgross I debugged the part of code on your original feature branch `AIC_BIC` and stored the input and output data. Then I created a test for the new function `getLLog()`. The results are the exactly the same, so now we have a single example.

All is fine and I am merging.

**Pull request merged**